app/views/contemporarypoets/items/browse_facet.html.erb

```diff
@@ -8,45 +8,65 @@
 <%# deviation from orchid default: moving the tabs to a partial for
     inclusion from other templates %>
 <%= render "contemporarypoets/browse_tabs" %>
+<% if @browse_facet != "person.name" %>
+  <div class="btn-group buffer-bottom-sm">
+    <button type="button" class="btn btn-default dropdown-toggle"
+      data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      <%= t "browse.sort_by", default: "Sort by" %>: <%= sort_selected_label("facets", sort_by) %>
+      <span class="caret"></span>
+    </button>
 
-<div class="btn-group buffer-bottom-sm">
-  <button type="button" class="btn btn-default dropdown-toggle"
-    data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-    <%= t "browse.sort_by", default: "Sort by" %>: <%= sort_selected_label("facets", sort_by) %>
-    <span class="caret"></span>
-  </button>
-
-  <ul class="dropdown-menu">
-    <% sort_fields_facets.each do |sort_field, label| %>
-      <%= render_overridable("sort_dropdown_item", label: label,
-        sort_url: prefix_path("browse_facet_path", facet_sort: sort_field)) %>
-    <% end %>
-  </ul>
-</div>
+    <ul class="dropdown-menu">
+      <% sort_fields_facets.each do |sort_field, label| %>
+        <%= render_overridable("sort_dropdown_item", label: label,
+          sort_url: prefix_path("browse_facet_path", facet_sort: sort_field)) %>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
 
 <div class="tab-content">
   <% @page_facets.each_with_index do |(facet_name, facet_info), index| %>
     <% id = facet_name.parameterize(separator: ".")  %>
     <div role="tabpanel"
       class="tab-pane <%= active?(facet_name, @browse_facet) %>" id="<%= id %>">
       <% if @res.key?(facet_name) %>
+        <%# switch for person.name? %>
+        <% if @browse_facet.include?"person.name" %>
+          <%= button_to "A-G", contemporarypoets_browse_facet_path("person.name_ag"), method: :get %>
```

Comment:
Later on we should make a few improvements to this https://github.com/CDRH/african_poetics/issues/283